src/search.js

```diff
@@ -3,8 +3,13 @@
 const emojilib = require('emojilib')
 const emojiNames = emojilib.ordered
 
-const verb = process.env.snippetapp ? 'Paste' : 'Copy'
-const preposition = process.env.snippetapp ? 'as snippet' : 'to clipboard'
+let verb = 'Copy'
+let preposition = 'to clipboard'
+
+const resetWordsForPasteByDefault = () => {
```

Comment:
I don't care for the side effects here. I think it would be simpler to just pass the correct phrases as parameters to `alfredItems`.

Reply:
Fair critique. I'd classify this more as a configuration option than a side effect, but they do certainly behave like global variables, so I'll plan to refactor this a bit.